### PR TITLE
[ENG-3542] Percy 3.0 upgrade

### PR DESCRIPTION
**Pull request by bp-cos**
-   Ticket: [ENG-3542]
-   Feature flag: n/a

## Purpose

Upgrade Percy from 1.x to 3.0

## Summary of Changes

Ran the `$ npx @percy/migrate` script and followed the prompts and did not update imports.

## Side Effects

None

## QA Notes

No testing needed

[ENG-3542]: https://openscience.atlassian.net/browse/ENG-3542?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -66,6 +66,13 @@ jobs:
     needs: build
     steps:
     - uses: actions/checkout@v2
+      - name: Percy Test
+        run: npx @percy/cli snapshot tests
+        env:
+          PERCY_PARALLEL_NONCE: ${{ env.PERCY_PARALLEL_NONCE }}
+          PERCY_PARALLEL_TOTAL: ${{ env.PERCY_PARALLEL_TOTAL }}
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+        timeout-minutes: 10
```
Comment (adlius) [inline .github/workflows/CI.yml:75]:
```suggestion
```

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -77,9 +77,8 @@ jobs:
         path: ./node_modules
         key: cached_node_modules_${{ secrets.CACHE_VERSION }}_${{ hashFiles('**/yarn.lock') }}
         restore-keys: cached_node_modules_${{ secrets.CACHE_VERSION }}_
-    - uses: percy/exec-action@v0.3.1
-      with:
-        custom-command: yarn test:ember --path ./dist --partition 1
+    - name: Run tests with percy snapshots
+      run: yarn percy exec --debug --parallel --verbose yarn test:ember --path ./dist --partition 1
```
Comment (adlius) [inline .github/workflows/CI.yml:81]: Now that it seems to be working, do we still need the `--debug` flag?

**Review by adlius — APPROVED** [review]
Looks good! One question before merging.

**Review by futa-ikeda — CHANGES_REQUESTED** [review]
the changes to percy look good, but now that this is targeting the basket branch, I think we need to rebase `feature/percy-upgrade` onto `basket/antimony`

**Review by adlius — APPROVED** [review]

**Review by futa-ikeda — APPROVED** [review]
As per scrum discussions, we're ok with leaving the debug flag in for now. Hoping later conversations with Percy will give us a better idea why it breaks without it.

**Pull request merged**